consensus/common/src/main/java/org/hyperledger/besu/consensus/common/PoaQueryServiceImpl.java

```diff
@@ -27,10 +28,15 @@
 
   private final BlockInterface blockInterface;
   private final Blockchain blockchain;
+  private final KeyPair keyPair;
```

Comment:
does it make sense to introduce `getLocalNodeKeyPair` method in either poaQueryService or poaMetricsService?

Reply:
, you have `getLocalSignerAddress` later on. You might get away with creating a default method in interface that calls getLocalNodeKeyPair method.

Reply:
not a show stopper though. Things looks OK as it is as well.

Reply:
I've updated the field name.